test/fixtures/dotenv/valid.env

```diff
@@ -33,3 +33,23 @@ RETAIN_INNER_QUOTES_AS_BACKTICKS=`{"foo": "bar's"}`
 TRIM_SPACE_FROM_UNQUOTED=    some spaced out string
 EMAIL=therealnerdybeast@example.tld
     SPACED_KEY = parsed
+
+MULTI_DOUBLE_QUOTED="THIS
+IS
+A
+MULTILINE
+STRING"
+
+MULTI_SINGLE_QUOTED='THIS
+IS
+A
+MULTILINE
+STRING'
+
+MULTI_BACKTICKED=`THIS
+IS
+A
+"MULTILINE'S"
+STRING`
+MULTI_NOT_VALID=THIS
```

Comment:
Can you add a test `MULTI_NOT_VALID='` like?

Reply:
Done